instrumentation/active_job/test/test_helper.rb

```diff
@@ -78,4 +78,5 @@ def initialize(*)
 OpenTelemetry::SDK.configure do |c|
   c.use 'OpenTelemetry::Instrumentation::ActiveJob'
   c.add_span_processor span_processor
+  c.logger = Logger.new(File::NULL)
```

Comment:
Is this meant to be in the PR?

Reply:
Yes. The diagnostic logs in the test case are unhelpful and noisy. It think we should make this the default for all tests.

Reply:
> It think we should make this the default for all tests.

I would support that if it were behind some kind of flag to turn it on or off globally (for tests). They are noisy and not usually useful, but then sometimes when the tests are broken you do want them. That tripped us up in https://github.com/open-telemetry/opentelemetry-ruby/pull/1136, for example.

Reply:
That would be OTEL_LOG_LEVEL 🤦

Reply:
Fixed in 9e429b11146a12dda68274fa63a428a242d52275

Reply:
Ty for explaining @arielvalentin. Setting `OTEL_LOG_LEVEL` seems reasonable to me!